playground/mocks/data/idp/idx/identify-unknown-user.json

```diff
@@ -52,7 +52,7 @@
     "type": "array",
     "value": [
       {
-        "message": "There is no account with the email <strong> test@rain.com </strong>. <a href=\"#\" class=\"js-sign-up\"> Sign up </a> for an account",
+        "message": "There is no account with the email test@rain.com. Error: idx.unknown.user",
```

Comment:
We have a [click handler](https://github.com/okta/okta-signin-widget/blob/f53b68168988903b0d63d4c584f68ef5693a03ee/src/v2/view-builder/views/IdentifierView.js#L155) that was listening to the click event from this anchor tag. If we are no longer supporting clicking on the link we should remove the handler as well.

Reply:
@nikhilvenkatraman-okta thanks for the review. The idea is to support click handler, but not with HTML tag, rather key off i18n key. See comment from @iancorcoran-okta  @pradeepdewda-okta from the accompanying https://github.com/okta/okta-core/pull/51872

Reply:
I haven't reviewed [this one](https://github.com/okta/okta-core/pull/51872#issuecomment-825786136) 

So we don't have to insert the error code in the message..  Following is enough to insert insert new error message, with link..
```
"i18n": {
          "key": "idx.unknown.user"
}
```